### ISSUE TITLE
Increased circulating gamma-delta T lymphocytes

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6066]
**Preferred term label:**
Increased circulating gamma-delta T lymphocytes

**Synonyms**
Increased gamma-delta T cells

**Definition (free text, please give PubMed ID)**
Increased number of gamma-delta t cells
PMID: 7923890

**Parent term (use hpo.jax.org/app)**
Abnormal T cell count

**Comment by pnrobinson** [comment]
This is https://hpo.jax.org/app/browse/term/HP:0500270
I will add the synonym
